website/content/en/v0.32/concepts/disruption.md

```diff
@@ -171,6 +171,11 @@ For Spot interruptions, the NodePool will start a new node as soon as it sees th
 Karpenter publishes Kubernetes events to the node for all events listed above in addition to __Spot Rebalance Recommendations__. Karpenter does not currently support taint, drain, and terminate logic for Spot Rebalance Recommendations.
 {{% /alert %}}
 
+{{% alert title="Note" color="warning" %}}
+For users that wish to benefit from both Karpenter and [Spot Rebalance Recommendations](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/rebalance-recommendations.html), which usually entails using the [AWS Node Termination Handler](https://github.com/aws/aws-node-termination-handler), please note that the AWS Node Termination Handler cordons and drains nodes, potentially causing node churn in the cluster. Further information can be found here: [Troubleshooting](../troubleshooting.md#aws-node-termination-handler-nth-interactions)
```

Comment:
I'd link the first mention of "Spot Rebalance Recommendations" in the previous alert/note, particularly if we agree to combine this alert with the previous one

Reply:
Sure, ill combine between them and submit new changes